pandas/_libs/tslib.pyx

```diff
@@ -86,7 +84,9 @@ import_array()
 cdef int64_t NPY_NAT = util.get_nat()
 iNaT = NPY_NAT
 
-from tslibs.timedeltas cimport parse_timedelta_string, cast_from_unit
+from tslibs.timedeltas cimport cast_from_unit, _delta_to_nanoseconds
+from tslibs.timedeltas import (Timedelta, convert_to_timedelta64,
+                               _delta_to_nanoseconds, array_to_timedelta64)
```

Comment:
double import

Reply:
why are you not cimporting?

Reply:
Only cimporting things that are used in `tslib`.  Other imports are pass-through for other modules to find in `tslib`.  Outside imports will be updated in a follow-up.

Reply:
_delta_to_nanoseconds cimport version is used in tslib, and python version is pass-through.

Reply:
what do you mean passthru? if its not used then remove it

Reply:
> Only cimporting things that are used in tslib. Other imports are pass-through for other modules to find in tslib. Outside imports will be updated in a follow-up.

no let's fix this here

Reply:
OK.  I'll de-privatize `_delta_to_nanoseconds` while I'm at it.